src/common.hpp

```diff
@@ -5,6 +5,7 @@
 #include <stdlib.h>
 #include <iostream>
 #include <sstream>
+#include <vector>
```

Comment:
Interesting; not now. (That is not the sort of thing I'd put in without error messages...)